### PR TITLE
added simple UTs with jest, yet to add enzyme based React tests

### PR DESCRIPTION
**Pull request by madhuv-msft**
with devops-ui not able to write cases, for the timebeing written cases for the non-devops component

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,14 +11,15 @@
...
+    "compile": "npm run clean && npm run compilesrc && npm run compiletest && npm run copy-all",
     "build": "npm run compile",
-    "build:dev": "npm run compile:dev && npm run postbuild",
     "copy-all": "npm run copy-scss && npm run copy-resources",
     "copy-scss": "copyfiles -u 1 src/**/*.scss dist",
-    "copy-resources": "copyfiles -u 1 src/**/Resources.js src/**/Types.d.ts dist"
+    "copy-resources": "copyfiles -u 1 src/**/Resources.js src/**/Types.d.ts dist",
+    "test": "npm run compiletest && jest ./dist_tests",
+    "compiletest": "tsc -p ./tests --outDir ./dist_tests",
+    "compilesrc": "tsc -p ./src --outDir dist"
```
Comment (savenkat) [inline package.json]: i think we are following the ./dest convention everywhere else

File: tests/Contracts/MockKubService.test.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { KubeServiceBase, KubeResourceType } from "../../src/Contracts/KubeServiceBase";
+
+class MockKubeService extends KubeServiceBase {
```
Comment (savenkat) [inline tests/Contracts/MockKubService.test.ts]: lets rename the file to KubeServiceBase.Tests.ts ?

File: tests/Contracts/MockKubService.test.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { KubeServiceBase, KubeResourceType } from "../../src/Contracts/KubeServiceBase";
+
+class MockKubeService extends KubeServiceBase {
+    public fetch(resourceType: KubeResourceType): Promise<any> {
+        return Promise.resolve(resourceType);
+    }
+}
+
+describe("KubeServiceBase Tests", () => {
+    let service: MockKubeService;
```
Comment (savenkat) [inline tests/Contracts/MockKubService.test.ts]: Shouldn't this be of type KubeServiceBase ?

Comment (madhuv-msft) [reply tests/Contracts/MockKubService.test.ts]: fixed

**Pull request merged**